data/remnant/remnant.txt

```diff
@@ -276,6 +276,490 @@ phrase "remnant"
+		"your target's steering"
+		"any engine"
+	word
+		" "
+	word
+		"and prevent them from escaping"
+		", and anyone won't stand a chance"
+	word
+		"."
+		"!"
+
+phrase "remnant thrasher cannon"
+	word
+		"a few"
+		"several"
+		"some"
+	word
+		" "
+	word
+		"thrasher cannons"
+		"thrasher guns"
+		"thrasher turrets"
+	word
+		"."
+	word
+		" "
+	word
+		"Compared to"
+		"When placed beside"
+	word
+		" "
+	word
+		"your energy blasters,"
+	word
+		" "
+	word
+		"they're more efficient in terms of heat"
+		"you can deal more damage with our cannons"
+		"they may be consuming more energy, but they have better heat and damage"
+		"our cannons may be more expensive, but they have better damage"
+		"they're larger, but they can deal more damage and release less heat"
+		"they're a force to reckoned with"
+	word
+		"."
+		"!"
+
+phrase "remnant emp torpedo bay"
+	word
+		"an EMP Torpedo Bay."
+	word
+		" "
+	word
+		"With a single shot, it"
+		"Firing one"
+		"One torpedo"
+	word
+		" "
+	word
+		"will"
+		"can"
+	word
+		" "
+	word
+		"disrupt"
+		"disable"
+		"deplete"
+	word
+		" "
+	word
+		"the enemy's systems"
+		"your target's batteries"
+		"your enemy's shields"
+		"the target's engines"
+	word
+		"."
+		"!"
+
+phrase "remnant salvage scanner"
+	word
+		"one or two"
+		"a dozen"
+		"some"
+	word
+		" "
+	word
+		"salvage scanners."
+	word
+		" "
+	word
+		"They combine outfit and tactical scanners into one package"
+		"Compared to your separated versions, our special scanner is more powerful"
+		"They have a longer range than your outfit scanner"
+		"Those babies scan fast"
+		"They're a must-have for plundering"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"Have you heard"
+		"Do you know"
+		"You know anything"
+	word
+		" "
+	word
+		"about"
+	word
+		" "
+	word
+		"the"
+		"those"
+	word
+		" "
+	word
+		"Quarg"
+		"Pug"
+		"Hai"
+		"Drak"
+		"Archons"
+		"insects far west of our space"
+	word
+		"?"
+	word
+		" "
+	phrase
+		"friendly remnant (aliens.!)"
+		"friendly remnant (aliens?)"
+
+phrase "friendly remnant (aliens.!)"
+	word
+		"They look strong"
+		"They look weak"
+		"Maybe we should contact them"
+		"I think they're dangerous"
+		"I think they're good"
+		"I think we can trust them"
+		"I want to plunder their tech"
+		"I want their ships"
+		"I haven't seen them yet"
+		"I wonder if the prefects already knew about them"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant (aliens?)"
+	word
+		"Are they strong"
+		"Are they weak"
+		"Are they friendly"
+		"Are they hostile"
+		"How strong are they"
+		"What are the odds of capturing their ships"
+		"Has your government met them"
+		"Is it worth trying to get my hands on their tech?"
+	word
+		"?"
+
+phrase "friendly remnant"
+	word
+		"Be careful"
+		"Stay alert"
+		"Be vigilant"
+	word
+		" "
+	word
+		"about"
+	word
+		" "
+	word
+		"the"
+	word
+		"Alphas in this galaxy"
+		"pirates in your space"
+		"Korath raiders"
+		"Korath"
+		"Archon in Nenia"
```

Comment:
I'd just remove this line as it doesn't really line up with the following words.
```suggestion
```